tests/data_context/test_concurrency_config.py

```diff
@@ -0,0 +1,31 @@
+from great_expectations.data_context import BaseDataContext
+from great_expectations.data_context.types.base import (
+    ConcurrencyConfig,
+    DataContextConfig,
+    InMemoryStoreBackendDefaults,
+)
+
```

Comment:
It might be trivial, but could we also add a test to demonstate that `concurrency` can be specified from the `great_expectations.yml`?

Reply:
I added `test_concurrency_enabled_with_yaml` below -- is this what you had in mind?

Reply:
Hmm - after seeing this, I guess it's probably not necessary. I was picturing something more like an integration test that loaded from a full yaml config, but we test things like that elsewhere, so it's not needed here.

Reply:
OK, I removed test_concurrency_enabled_with_yaml